css/includes/components/_utils.scss

```diff
@@ -44,13 +44,6 @@
     transition: border-color 0.3s;
 }
 
-// Apply the same font-size and line-height as btn-text.
-// Useful when replacing a btn-text content by a loading icon.
-.btn-text-loading {
-    font-size: var(--tblr-btn-font-size) !important;
-    line-height: var(--tblr-btn-line-height) !important;
-}
-
```

Comment:
This class and its usages should not be removed.

After your changes:
![image](https://github.com/user-attachments/assets/2af7e40c-075f-4adb-bf6f-76b495ee33a7)

On main:
![image](https://github.com/user-attachments/assets/d72e4a69-c48d-4a68-869f-43c44c372a70)

Reply:
Please test with the latest commit